### ISSUE TITLE
all videos from youtube not working 

### ISSUE THREAD

**Issue by kyrellous** [HaarigerHarald/android-youtubeExtractor#218]
videos came to an error 
"D/extractor : response not found"

**Comment by candratop3** [comment]
same here...new browser

youTubeExtractor: Extraction failed

 private static final String USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.98 Safari/537.36";

**Comment by Demerro** [comment]
That because it trying to open modified by extractor link, for example 

> https://www.youtube.com/get_video_info?video_id=8MLa-Lh8lkU&eurl=https%3A%2F%2Fyoutube.googleapis.com%2Fv%2F8MLa-Lh8lkU

And the page is not available. I guess youtube changed something.

**Stack trace:**

W/System.err: java.io.FileNotFoundException: https://www.youtube.com/get_video_info?video_id=8MLa-Lh8lkU&eurl=https%3A%2F%2Fyoutube.googleapis.com%2Fv%2F8MLa-Lh8lkU
        at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:255)
        at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getInputStream(DelegatingHttpsURLConnection.java:211)
        at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:30)
        at at.huber.youtubeExtractor.YouTubeExtractor.getStreamUrls(YouTubeExtractor.java:219)
        at at.huber.youtubeExtractor.YouTubeExtractor.doInBackground(YouTubeExtractor.java:195)
W/System.err:     at at.huber.youtubeExtractor.YouTubeExtractor.doInBackground(YouTubeExtractor.java:34)
        at android.os.AsyncTask$3.call(AsyncTask.java:394)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
        at java.lang.Thread.run(Thread.java:923)

**Comment by Vishal-beep136** [comment]
Same to Same I'm also facing this issues

**Comment by JMerchanFL** [comment]
Hi,

I've changed line 61 and updated Chrome version to 99.0.4844.51 now it works perfectly.

Greetings.

**Comment by TaslimOseni** [comment]
@JMerchanFL, what is line 61? I'm currently facing this issue. [This](https://www.youtube.com/watch?v=iX_BkRlro4Y) is an example of a URL that is giving the said exception.

My conjecture is that YouTube definitely changed something.

**Comment by TaslimOseni** [comment]
@JMerchanFL, what is line 61? I'm currently facing this issue.

**Comment by JMerchanFL** [comment]
This one: 

https://github.com/HaarigerHarald/android-youtubeExtractor/blob/c9044efa3a0a986d85e13bc48d64377cc58d2cf5/youtubeExtractor/src/main/java/at/huber/youtubeExtractor/YouTubeExtractor.java#L60

**Comment by JosiYosi** [comment]
> That because it trying to open modified by extractor link, for example
> 
> > https://www.youtube.com/get_video_info?video_id=8MLa-Lh8lkU&eurl=https%3A%2F%2Fyoutube.googleapis.com%2Fv%2F8MLa-Lh8lkU
> 
> And the page is not available. I guess youtube changed something.
> 
> **Stack trace:**
> 
> W/System.err: java.io.FileNotFoundException: https://www.youtube.com/get_video_info?video_id=8MLa-Lh8lkU&eurl=https%3A%2F%2Fyoutube.googleapis.com%2Fv%2F8MLa-Lh8lkU at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:255) at com.android.okhttp.internal.huc.DelegatingHttpsURLConnection.getInputStream(DelegatingHttpsURLConnection.java:211) at com.android.okhttp.internal.huc.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:30) at at.huber.youtubeExtractor.YouTubeExtractor.getStreamUrls(YouTubeExtractor.java:219) at at.huber.youtubeExtractor.YouTubeExtractor.doInBackground(YouTubeExtractor.java:195) W/System.err: at at.huber.youtubeExtractor.YouTubeExtractor.doInBackground(YouTubeExtractor.java:34) at android.os.AsyncTask$3.call(AsyncTask.java:394) at java.util.concurrent.FutureTask.run(FutureTask.java:266) at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:305) at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167) at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641) at java.lang.Thread.run(Thread.java:923)

Yes, they updated Chrome to a new version. That is why this error shows. Do the following in YoutubeExtractor java class

Old one:
`private static final String USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/40.0.2214.115 Safari/537.36";`

Change to a new one:

`private static final String USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.98 Safari/537.36";`

Basically as you can see, Chrome is updated to Chrome/97.0.4692.98. Safari is not changed yet.

**Comment by JosiYosi** [comment]
> videos came to an error "D/extractor : response not found"

Old one:
`private static final String USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/40.0.2214.115 Safari/537.36";
`
Change to a new one:

`private static final String USER_AGENT = "Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.98 Safari/537.36";`

Basically as you can see, Chrome is updated to Chrome/97.0.4692.98. Safari is not changed yet.

**Comment by lukaskris** [comment]
it not works now
